google/cloud/security/common/data_access/csv_writer.py

```diff
@@ -385,6 +385,15 @@
     'iam_policy'
 ]
 
+SERVICE_ACCOUNTS_FIELDNAMES = [
+    'project_id',
+    'name',
+    'email',
+    'oauth2_client_id',
```

Comment:
+100 for this data; I think it will be really useful for tracking the permissioning of gsuite's api scopes, which right now is assigned by client id, but is really hard to determine what service account it's for.